prefetch.js

```diff
@@ -109,16 +107,21 @@
       return true;
     }
 
+    function createLinkTag(url){
+      var link = document.createElement('link');
+      link.setAttribute('rel', 'prefetch');
+      link.setAttribute('href', url);
+      return link;
+    }
+
     function injectPrefetchLink(a){
-      if(a && isPrefetchable(a)){
-        var url = (typeof a === 'object') ? a.href : a;
-        var link = document.createElement('link');
-        link.setAttribute('rel', 'prefetch');
-        link.setAttribute('href', url);
+      var url = (typeof a === 'object') ? a.href : a;
+      var link = (url) ? createLinkTag(url) : null;
+      if(link){
         document.getElementsByTagName('head')[0].appendChild(link);
```

Comment:
Looks like some version of [IE has issues](http://stackoverflow.com/questions/10159130/document-head-appendchild-or-document-createelement-not-working-in-ie) with doing it that way. Wish I could do it that way, but I think I'll leave it due to this.